src/Components/SearchError/SearchError.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import '../SearchError/_SearchError.scss'
+
+type Props = {
+  errorMessage: string;
+}
+
+const SearchError = ({ errorMessage } : Props) => {
+  return (
+    <div className='search-error-area'>
+      <h3 className='search-error-message'>{errorMessage}</h3>
+    </div>
+  )
```

Comment:
I like how you broke this into its own component for easier reading in `SearchForm`